components/app/AudioPlayer.vue

```diff
@@ -551,6 +551,9 @@ export default {
     timeupdate() {
       if (!this.$refs.playedTrack) {
         console.error('Invalid no played track ref')
+        this.pause()
+        this.closePlayback()
+        // TODO: Attempt to rehydrate Vue session without interrupting native playback.
```

Comment:
```suggestion
        this.closePlayback()
```

Pause is unnecessary here and I don't think we should rehydrate here. As I mentioned in https://github.com/advplyr/audiobookshelf-app/issues/1385#issuecomment-3568360052 the only time this should happen is when webview is first mounted (with the exception of the disconnect bug). I believe the major issue is devices killing webview in the background, so when the user opens the app again it mounts webview again. Then we can implement the attempt to sync Vue/Native.